esp-hal/src/dma/mod.rs

```diff
@@ -397,6 +399,24 @@ macro_rules! dma_circular_descriptors {
     };
 }
 
+/// Allocates a DMA buffer with a specific size, aligned to 4 bytes
+#[doc(hidden)]
+#[macro_export]
+macro_rules! declare_aligned_dma_buffer {
+    ($name:ident, $size:expr) => {
+        static mut $name: [u32; ($size + 3) / 4] = [0; ($size + 3) / 4];
+    };
+}
+
+/// Allocates a DMA buffer with a specific size, aligned to 4 bytes
+#[doc(hidden)]
+#[macro_export]
+macro_rules! as_bytes_mut {
+    ($name:ident, $size:expr) => {
+        unsafe { core::slice::from_raw_parts_mut($name.as_mut_ptr() as *mut u8, $size) }
```

Comment:
You could cast this to `[u8; $size]` to avoid the breaking change but idk how big a deal it is.

Reply:
Thanks, good call